bindings/java/src/org/sleuthkit/datamodel/TimelineManager.java

```diff
@@ -501,7 +511,7 @@ private long addEventDescription(long dataSourceObjId, long fileObjId, Long arti
 
 			for (Map.Entry<TimelineEventType, Long> timeEntry : timeMap.entrySet()) {
 				Long time = timeEntry.getValue();
-				if (time > 0) {// if the time is legitimate ( greater than zero ) insert it
+				if (time > 0 || time <= MAX_TIMESTAMP_TO_ADD) {// if the time is legitimate ( greater than zero or less then 12 years from current date) insert it
```

Comment:
@markmckinnon and @raman-bt, shouldn't this be an AND test? With short-circuit evaluation of the conditional, all times greater than zero will pass the test.

Reply:
You're right Richard. Ugh, I missed that.

Reply:
Fixed